### ISSUE TITLE
Test epic

### ISSUE THREAD

**Issue by oysteinmoseng** [highcharts/highcharts#10877]
test

**Comment by cvasseng** [comment]
###### do-not-edit-start-codetree-epic-issues

#### Issues in this epic:

Title | Milestone | Assignees | Stage | State
----- | --------- | --------- | ----- | -----
![](https://cdnjs.cloudflare.com/ajax/libs/octicons/4.4.0/svg/issue-opened.svg) [Use empty aria-label for series that has no label #10876](https://github.com/highcharts/highcharts/issues/10876) | N/A | N/A | Open


###### do-not-edit-end-codetree-epic-issues
